docs/src/docs/browsercaps/common-browser-caps.adoc

```diff
@@ -23,19 +23,27 @@ WEB_DRIVER_MANAGER.setGlobalCapability(FirefoxOptions.FIREFOX_OPTIONS, options);
 
 ====
 
+= Request capabilities
 
-== Request capabilities
-
-Some WebDriverRequests support setting capabilities, like the `DesktopWebDriverRequest`.
+A request defined capability means using a Webdriver request like `DesktopWebDriverRequest`.
 
+.Set capabilities to a DesktopWebDriverRequest object
 [source,java]
 ----
-WebDriverRequest myRequest = new DesktopWebDriverRequest();
+DesktopWebDriverRequest request = new DesktopWebDriverRequest();
+DesiredCapabilities caps = request.getDesiredCapabilities();
 myRequest.getDesiredCapabilities().put(CapabilityType.ACCEPT_INSECURE_CERTS, true);
-WebDriver webDriver = WEB_DRIVER_MANAGER.getWebDriver(myRequest);
+// Start your session with the DesktopWebDriverRequest object
+WebDriver driver = WebDriverManager.getWebDriver(request);
 ----
 
-== User agent configuration
+[NOTE]
+=====
+Have a look into <<Browser specific knowledge>> for specific browser options. +
```

Comment:
Anchor `Browser specific knowledge` does not exist? (And also did not seem to exist previously, so unsure whence it should point.)

Reply:
The links points to chapter in `browser-specific-knowledge.adoc`.

Reply:
It points to `https://github.com/telekom/testerra/blob/docs/add-links-to-tt-version-docs/docs/src/docs/browsercaps/common-browser-caps.adoc#Browser%20specific%20knowledge` though.